woocommerce/woocommerce-android
# [Backed Receipt] Download and share a file not a link

Closes: #10648
Closes: #10625 (now clear that we target 8.7.0 as WC version that will have backend generated receipts)
Closes: #10632 (now clear that 2 days of expiration days will be enough)


### Description

Before this PR we shared the receipt
* Only via email intended
* We attached a link

No, we share in a generic way so it can be sent via any supported app (e.g., messengers), and we share a file, which is downloaded by a given link. File sharing is needed to reduce the amount of issues in cases when a link on a receipt expires.

### Testing instructions

Test on both the release version of WC and the dev version that has a backend receipt generation
* Open a order that has a receipt (all paid orders have receipts if the backend supports receipt generation)
* Click "See receipt" -> Send.
* Notice that you can share it via different apps and that we attach file now

* Collect payment via a card reader
* On the last step click on "send receipt"
* Notice that you can share it via different apps and that we attach file now

### Images/gif



https://github.com/woocommerce/woocommerce-android/assets/4923871/0a450e9e-976b-454c-b06d-f7503636bc63
